app/src/main/res/layout/activity_repository.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
```

Comment:
Is this LinearLayout really necessary? Or should the RecyclerView be the direct root of this layout file?